### ISSUE TITLE
[Apex] Duplicate warning for EmptyStatementBlock and EmptyifStmt

### ISSUE THREAD

**Issue by stefanoportelli** [pmd/pmd#3164]
**Affects PMD Version:** 
PMD 6.32

**Description:**

With a class that contains code like this:

```
[...]
        if( processCasesMap.containsKey('C_RT_TakeoverEle') ) {}
[...]
```

PMD will report 2 warnings, one for **EmptyStatementBlock** the other for **EmptyIfStmt** on the same line

**Code Sample demonstrating the issue:**

[TestClass.cls.txt](https://github.com/pmd/pmd/files/6155288/TestClass.cls.txt)

is reporting duplicate warning on the following lines:
![image](https://user-images.githubusercontent.com/26385026/111448035-562cca80-870e-11eb-96c3-4c073512ca3c.png)

**Steps to reproduce:**

1. Download the TestClass.cls.txt file
2. Rename the file to TestClass.cls
3. run PMF 6.32 with [apex_ruleset.xml.txt](https://github.com/pmd/pmd/files/6155305/apex_ruleset.xml.txt)

**Running PMD through:** *[CLI]*

**Comment by oowekyala** [comment]
Thanks for the report. There's an identical ticket for the java rules (#3096). Like in the java module, I think we should just make a single rule that flags empty blocks and statements, and remove EmptyIfStmt and the others, which would be covered by this single rule. Relevant rules: [EmptyCatchBlock](https://pmd.github.io/latest/pmd_rules_apex_errorprone.html#emptycatchblock), EmptyIfStmt, EmptyStatementBlock, EmptyTryOrFinallyBlock, EmptyWhileStmt
